lib/shoulda/matchers/active_record.rb

```diff
@@ -18,6 +18,7 @@
 require "shoulda/matchers/active_record/define_enum_for_matcher"
 require "shoulda/matchers/active_record/uniqueness"
 require "shoulda/matchers/active_record/validate_uniqueness_of_matcher"
+require "shoulda/matchers/active_record/errors"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.